test/system/252-quadlet.bats

```diff
@@ -1699,4 +1699,31 @@ EOF
         fi
     done < <(parse_table "${dropin_files}")
 }
+
+# Following issue: https://github.com/containers/podman/issues/24599
+# Make sure future changes do not break
+@test "quadlet - build with pull" {
```

Comment:
This is going to cause flakes. `buildah` cannot run in parallel, hence `podman build` cannot run in parallel: https://github.com/containers/buildah/issues/5674

In my foolish youth, I overoptimistically added the `ci:parallel` tag to this entire file. That will apply here, which will cause this test to flake. 

Proposed solution(?): override `file_tags`, or move this test into a separate file. I don't know which is better. To override, [it looks like you can redefine file_tags](https://bats-core.readthedocs.io/en/stable/writing-tests.html#tagging-tests), then redefine it again below this test. That's barfy. So is a separate file. There might be better solutions that I can't think of right now.

Reply:
Will this cause flakes even where there no layers? The Containerfile is just `FROM xxx` so is this already enough to trigger flakes?

Reply:
I honestly don't know. I've never dived deeply enough into that failure. Maybe it's OK to leave this as it is, and play the wait-and-see game.

Reply:
@edsantiago I rather not have to split the files. But, I don't mind clearing the flag and setting it again around the test. Do you think it's safer? No point in running later to fix it

Reply:
I'm really sorry... I don't know. I've never actually tried the change-file-tags trick. It seems worth a try. It also might be okay to merge as-is and see if it flakes or not: that might provide insight into the nature of the buildah-parallel bug.

Reply:
So, if you're OK with it, I'll keep it this way and we'll see what happens. Thanks